cmd/http/listener.go

```diff
@@ -230,6 +234,7 @@ func (listener *httpListener) start() {
 					reqInfo.AppendTags("localAddr", bufconn.LocalAddr().String())
 					ctx := logger.SetReqInfo(context.Background(), reqInfo)
 					logger.LogIf(ctx, err)
+					bufconn.Write([]byte(malformedErrMsgFn(err)))
```

Comment:
The decision was `close the connection for invalid HTTP requests` and no need to write error message

Reply:
A user already requested this @balamurugana that we are closing connections without sending an error back to the client. I don't remember if that was decided...